Tests/Resources/bin/runtests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "<?php exit(defined('HHVM_VERSION') ? 1 : 0);" > /tmp/ishhvm.php
+
+php /tmp/ishhvm.php
```

Comment:
see for example
https://github.com/symfony-cmf/RoutingBundle/blob/master/.travis.yml#L37

   - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then